### DISCUSSION TITLE
How can I export a flow created by me to someone else on my team?

### DISCUSSION THREAD

**Discussion by warleyvods-meli** [postmanlabs/postman-flows#113] [Q&A] [answered]
Hi, how can I export a flow created by me to someone else on my team?
I tried to export using a link that postman provides, but it didn't work.

**Comment by samay-v** [comment, marked as answer]
Hey @warleyvods-meli. You can refer to https://github.com/postmanlabs/postman-flows/discussions/35#discussioncomment-1496835 to understand how to collaborate on a flow
